docs/PropTypes/Async.ts

```diff
@@ -1,10 +1 @@
-// import { Component } from 'react';
-//
-// import { AsyncProps } from 'react-select/src/Async';
-// import { GroupBase, OptionBase } from 'react-select';
-//
-// export default class Select<
-//   Option extends OptionBase,
-//   IsMulti extends boolean,
-//   Group extends GroupBase<Option>
-// > extends Component<AsyncProps<Option, IsMulti, Group>> {}
+export { AsyncAdditionalProps } from 'react-select/src/useAsync';
```

Comment:
magical-types resolves the type signature down through interface extensions. 
We only pull in AsyncAdditionalProps here to make the docs experience nicer.